### PR TITLE
Add: Icon, Image, Input, Text and Touchable components

### PR DESCRIPTION
**Pull request by jpirazusta**
### Pull request type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): New components

---

### Description

Adds:
- Icon
- Image
- Input
- Text
- Touchable

### CODE REVIEW

**Review by ManuViola77 — APPROVED** [review]
👏🏻

File: src/design-system/components/Icon/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { Image as DripsyImage, View } from 'dripsy';
+
+import { icons } from '../../assets';
+import { IconProps } from '../Input/types';
+import { Touchable } from '../Touchable';
+import styles from './styles';
```
Comment (nicmalegre) [inline src/design-system/components/Icon/index.tsx:8]: WDYT to use absolute imports here?

File: src/design-system/components/Image/index.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, { ComponentProps } from 'react';
+
+import { Image as DripsyImage, SxProp } from 'dripsy';
+
+import { images } from '../../assets';
```
Comment (nicmalegre) [inline src/design-system/components/Image/index.tsx:5]: Same here

**Review by nicmalegre — APPROVED** [review]

File: src/design-system/components/Image/index.tsx (hunk 2 above)
Comment (jpirazusta) [reply src/design-system/components/Image/index.tsx:5]: Ok, cool! I can change in the next PR, if it's ok. Since I modified the config for the paths

**Pull request merged**